src/index.html

```diff
@@ -22,13 +22,12 @@
     <main id="container"></main>
     <!-- The Modal -->
     <div id="commentModal" class="modal">
-
       <!-- Modal content -->
-       <div class="modal-content">
+      <div class="modal-content">
         <span class="close">&times;</span>
-         <div id="modalContent"></div>
-       </div>
+        <div id="modalContent"></div>
       </div>
+    </div>
```

Comment:
Hi @luis-pomare  in these lines of code you have duplicate elements, this could cause you some errors or unwanted views.